### PR TITLE
WTDD: Update UI api docs

### PR DESCRIPTION
**Pull request by DingoEatingFuzz**
These were very out of date, lol.

### CODE REVIEW

**Review by backspace — CHANGES_REQUESTED** [review]
whew, so much new material! I wonder if we can embed this in the code somehow in the future 🤔

I made a couple of spelling fix suggestions and maybe `page` has the wrong default but otherwise 👍😀

File: website/pages/api-docs/ui.mdx [hunk 1]
```diff
@@ -112,11 +119,49 @@ in a table to drill into for task events.
...
+- `sort` `(string: "")` - Specifies the property the list of allocations should be sorted by.
+  This is specified as a query string parameter.
+
+- `desc` `(boolean: false)` - Specifies whether or not the sort direction is descending
+  or ascending. This is specified as a query string parameter.
+
+- `search` `(string: "")` - Specifies a regular expression used to filter the list of
+  visible allocations. This is specified as a query string parameter.
+
+- `page` `(int: 0)` - Specifies the page in the allocations list that should be visible. This
```
[the remark this answers is not in the document]
Comment (DingoEatingFuzz) [reply website/pages/api-docs/ui.mdx:141]: Good catch on this; I can update it throughout.

**Review by backspace — APPROVED** [review]
total success 🥳

**Comment by backspace** [comment]
or not, I see there’s a merge conflict now 😞

**Comment by DingoEatingFuzz** [comment]
I expected the merge conflicts. The `next-mdx-remote` migration merged yesterday. I'll get on the rebase.

**Pull request merged**